tests/core/pyspec/eth2spec/test/helpers/block.py

```diff
@@ -126,6 +126,11 @@ def build_empty_block(spec, state, slot=None, proposer_index=None):
     if is_post_bellatrix(spec):
         empty_block.body.execution_payload = build_empty_execution_payload(spec, state)
 
+    if is_post_electra(spec):
+        empty_block.body.requests.deposit_requests = []
```

Comment:
Aren’t they empty by default?

Reply:
Tests were not happy when I did not initialize them as empty lists 😅